packages/labs/ssr-react/src/lib/node/wrap-create-element.ts

```diff
@@ -19,49 +19,50 @@ import type {
  * Wraps the provided `createElement` function to also server render Lit
  * component's shadow DOM.
  */
-export function wrapCreateElement(
+export const wrapCreateElement = (
   // TODO(augustjk) Should the type for this param be more generic to allow
   // non-React alternatives like preact?
   originalCreateElement: typeof ReactCreateElement
-) {
-  return function litPatchedCreateElement<P>(
-    type: ElementType<P>,
-    props: PropsWithChildren<P> | null,
-    ...children: ReactNode[]
-  ): ReactElement {
-    if (isCustomElement(type)) {
-      const {shadowContents, elementAttributes, templateAttributes} =
-        renderCustomElement(type, props);
+) =>
+  originalCreateElement.name !== 'litPatchedCreateElement'
```

Comment:
Thank you! I agree, short path first is better readability. Opted to remove the ternary implicit return and do an explicit early return for the short path.

The `if (isCustomElement(...))` is less amenable to this because there's another condition check after rendering the custom element, for which the final return is a fallback for so can't early return with that.

Reply:
Ended up removing the name check. But still moved to explicit return over arrow function implicit return. Seems more readable that way.